src/selectors/Entities/name.js

```diff
@@ -85,6 +85,10 @@ export const selectVaccinePatientHistory = patient => {
     "code == 'vaccination'"
   );
   const { id: vaccinationPatientEventID } = vaccinationPatientEvent ?? {};
+  const [supplementalDataSchema = {}] = selectSupplementalDataSchemas();
+
+  const hasDateOfVaccinationInSchema =
+    supplementalDataSchema?.jsonSchema?.properties?.dateOfVaccination;
```

Comment:
safest to convert to a boolean, since that's the expected var type
```suggestion
  const hasDateOfVaccinationInSchema =
    !!supplementalDataSchema?.jsonSchema?.properties?.dateOfVaccination;
```